### PR TITLE
employee agenda client name url fix TM-2976

### PR DESCRIPTION
**Pull request by SidelineCory24**
[FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2151)

### CODE REVIEW

**Comment by SidelineCory24** [comment]
I was able to get create the same URL on the API side and DEV1 but still did not see the results filter out. Not sure if this is a local/mock issue.

**API URL**
http://host.docker.internal:3333/v1/tm-persons?rp.pageNum=0&rp.pageRows=1&rp.filter=tmperperdetseqnum%7CEQ%7C8364589%7C

**DEV1 URL**
https://gtmdev1hronline.hr.dosdev.us/Dev1/HRTMAPData/api/v1/tm-persons?rp.pageRows=1&rp.pageNum=0&rp.filter=tmperperdetseqnum%7Ceq%7C383044%7C

File: talentmap_api/fsbid/urls/agenda_employees.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 urlpatterns = [
     url(r'^$', views.FSBidAgendaEmployeesListView.as_view(), name="agenda-employees-FSBid-agenda-employees-list"),
+    url(r'^employee/(?P<pk>[0-9]+)/$', views.FSBidAgendaEmployeesEmployeeListView.as_view(), name="agenda-employees-FSBid-agenda-employees-employee"),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/urls/agenda_employees.py:10]: might not need a new URL

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 2]
```diff
@@ -205,6 +234,27 @@ def convert_agenda_employees_query(query):
...
+
+def convert_agenda_employee_query(perdet, query):
+    '''
+    Convert TalentMAP filter into FSBid filter for one employee
+    '''
+    filters = [
+        {"col": "tmperperdetseqnum", "com": "EQ", "val": perdet},
+    ]
+
+    filters = pydash.filter_(filters, lambda o: o["val"] != None)
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:250]: what's the motivation behind this line?

File: talentmap_api/fsbid/urls/agenda_employees.py (hunk 1 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/urls/agenda_employees.py:10]: discussed offline, will use a name placeholder instead and will update properly once further integration is done

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 2 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda_employees.py:250]: my inability to properly cleanup after testing, code has been removed

File: talentmap_api/fsbid/services/agenda.py [hunk 3]
```diff
@@ -56,7 +57,14 @@ def get_agenda_items(jwt_token=None, query = {}, host=None):
         **args
     )
 
-    return agenda_items
+    perdet = query['perdet']
+    employeeQuery = { "limit": 1 }
+    employee = get_agenda_employee(employeeQuery, perdet, jwt_token, host)
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:62]: is there a reason for sending perdet and limit separate?

Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:62]: is there a reason for sending perdet and limit separate?

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda.py:62]: no, just bad cleanup again. I originally was sending the `perdet` through from the FE to the url, but didn't update the function params completely once I made the last set of changes. Update made.

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 4]
```diff
@@ -205,10 +206,11 @@ def convert_agenda_employees_query(query):
         values["rp.pageNum"] = 0
         values["rp.pageRows"] = 0
         values["rp.columns"] = "ROWCOUNT"
-
+
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:209]: 🤔

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 5]
```diff
@@ -205,10 +206,11 @@ def convert_agenda_employees_query(query):
         values["rp.pageNum"] = 0
         values["rp.pageRows"] = 0
         values["rp.columns"] = "ROWCOUNT"
-
+    
     valuesToReturn = pydash.omit_by(values, lambda o: o is None or o == [])
     return urlencode(valuesToReturn, doseq=True, quote_via=quote)
 
+
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:213]: 💭

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 6]
```diff
@@ -164,6 +164,7 @@ def convert_agenda_employees_query(query):
         {"col": "tmperhsorgcode", "com": "IN", "val": query.get("handshake-organizations", None)},
         {"col": "tmpercdoid", "com": "IN", "val": query.get("cdos", None)},
         {"col": "tmperperscode", "com": "IN", "val": "S,L,A,P,U"},
+        {"col": "tmperperdetseqnum", "com": "EQ", "val": query.get("perdet", None)},
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:167]: 🙌

**Comment by elizabeth-jimenez** [comment]
@SidelineCory24 just two formatting things to address - good job!

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 4 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda_employees.py:209]: 🤷‍♂️

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 5 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda_employees.py:213]: 🤷‍♂️

**Pull request merged**